support/cas-server-support-oauth-core-api/src/main/java/org/apereo/cas/support/oauth/web/response/callback/OAuth20TokenAuthorizationResponseBuilder.java

```diff
@@ -57,10 +57,17 @@ public ModelAndView build(final JEEContext context,
         val accessToken = result.getAccessToken().orElse(null);
         val refreshToken = result.getRefreshToken().orElse(null);
         LOGGER.debug("Generated OAuth access token: [{}]", accessToken);
+        OAuth20JwtAccessTokenEncoder.builder()
+            .accessToken(accessToken)
+            .registeredService(holder.getRegisteredService())
+            .service(holder.getService())
+            .accessTokenJwtBuilder(accessTokenJwtBuilder)
+            .casProperties(casProperties)
+            .build()
+            .encode();
         return buildCallbackUrlResponseType(holder, redirectUri, accessToken, new ArrayList<>(0), refreshToken, context);
     }
 
-
```

Comment:
The move of OAuth20AccessTokenEncoder.builder() is to fix a null point exception in OidcImplicitIdTokenAuthorizationResponseBuilder (subclass of OAuth20TokenAuthorizationResponseBuilder) caused by the commit 5cbe46c. This commit expects OAuth20JwtAccessTokenEncoder.encode() to be called before OidcIdTokenGeneratorService.generate().